plugins/zynaddsubfx/zynaddsubfx/style.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
```

Comment:
I don't believe we use this file at all and it's not our code.  This change is unnecessary.